strawberryfields/result.py

```diff
@@ -73,6 +80,15 @@ def __init__(self, result: Mapping, ancilla_samples: Mapping = None) -> None:
         self._result = result
         self._ancilla_samples = ancilla_samples
 
+    @property
+    def raw(self) -> Mapping:
+        """The raw results dictionary containing any samples and metadata.
+
+        Returns:
+            dict: samples and metadata
+        """
+        return self._result
```

Comment:
I was thinking about whether it would be better not to return the results directly like this, but rather copy it or simply return a view of the dictionary, to avoid users accidentally changing anything in the `Result` object. Not sure if it's worth it or, in that case, how to best do that.

Reply:
Sure, I agree. 👍

Reply:
Is there a clear use case for accessing the results like this?

Reply:
Maybe not. It'd just be a way of easily getting direct access to the results obtained from XCC. 🤔

Reply:
Perhaps it could be useful for debugging? But then again, users can just access `_result` directly.

Reply:
This :point_up: was exactly my thinking

Reply:
Perhaps we should just remove it then? I see no real reason to have it, to be honest, since there will probably not be anything in `Result.raw` that isn't already provided via `Result.samples` or `Result.metadata`. 🤔